README.md

```diff
@@ -987,6 +987,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Emmy Steven](https://github.com/emmysteven)
   - [Emore Ogheneyoma Lawrence](https://github.com/devyoma)
   - [Enn Dee](https://github.com/enn-dee)
+  - [Enikuomehin David](https://github.com/eniiku)
```

Comment:
just swap the name according to the dictionary order

Reply:
Thanks for pointing it out for me.